seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/io/seata/spring/boot/autoconfigure/SeataTCCFenceAutoConfiguration.java

```diff
@@ -38,7 +37,7 @@
 @ConditionalOnExpression("${seata.enabled:true}")
 @ConditionalOnBean(type = {"javax.sql.DataSource", "org.springframework.transaction.PlatformTransactionManager"})
 @ConditionalOnMissingBean(TCCFenceConfig.class)
-@AutoConfigureAfter({SeataCoreAutoConfiguration.class, DataSourceAutoConfiguration.class, DataSourceTransactionManagerAutoConfiguration.class})
+@AutoConfigureAfter({SeataCoreAutoConfiguration.class, DataSourceAutoConfiguration.class})
```

Comment:
Can I understand that actually SeataTCCFenceAutoConfiguration does not need to be specifically specified to be loaded after DataSourceTransactionManagerAutoConfiguration, just make sure that PlatformTransactionManager bean exists?

Reply:
Yes, otherwise the transaction manager will not work if the project uses spring data JPA.

Reply:
Similarly, please delete `DataSourceAutoConfiguration.class`.

Reply:
Removed.